routemodel/data_retrieval/get_navigation.py

```diff
@@ -141,57 +105,15 @@ def parse_route_data(self, response):
             return False
     
         # write dataframe to csv
-        route_df.to_csv(ROUTES_FILE)
+        route_df.to_csv(NAVIGATION_FILE)
         print(f'Routes data written to CSV file.')
         return True
 
-    def parse_elevation_data(self, response):
-        """
-        Parsing through Elevations API call response.
-        @param response: Requests.response object from API call
-        @return: boolean
-        """
-        # convert response into dict
-        elevations = response.json()
-        # parse down to relevant information
-        elevations = elevations['resourceSets'][0]['resources'][0]['elevations']
-
-        # create DataFrame with headers
-        headers = ['Latitude', 'Longitude', 'Elevation']
-        elevation_df = pd.DataFrame(columns=headers)
-
-        # loop through waypoints and viawaypoints
-        # to write coordinates into DataFrame
-        counter = 0
-        for index, waypoint in enumerate(self.waypts):
-            for key in waypoint.keys():
-                elevation_df = elevation_df.append({'Latitude': key,
-                                                    'Longitude': waypoint[key],
-                                                    'Elevation': elevations[counter]}, ignore_index = True)
-                counter += 1
-            try:
-                for viawaypoints in self.viawaypts[index]:
-                    for key in viawaypoints.keys():
-                        elevation_df = elevation_df.append({'Latitude': key,
-                                                            'Longitude': viawaypoints[key],
-                                                            'Elevation': elevations[counter]}, ignore_index = True)
-                    counter += 1
-            except IndexError:
-                pass
-
-
-        # write dataframe into csv and return true
-        elevation_df.to_csv(ELEVATIONS_FILE)
-        print(f'Elevations written to CSV file.')
-        return True
-
-
 if __name__ == "__main__":
```

Comment:
Ideally we remove this main that executes code at the bottom of it, and incorporate its functionality into a test :)